### DISCUSSION TITLE
2月升级到专业版 突然变为免费版

### DISCUSSION THREAD

**Discussion by by2308333** [getlantern/lantern-forum-cn#538] [Q&A]
2月升级到专业版 之前发现登录账号在多个账号之间变换 现在变为了免费版 重新通过邮箱615985536@qq.com授权，但收不到恢复码
!
![Uploading 蓝灯截图.png…]()

**Comment by MaxMKMax** [comment]
@by2308333 好的. 我帮您提交以下问题. 谢谢!

**Comment by by2308333** [comment]
什么时候能有结果啊

**Comment by by2308333** [comment]
还是得感谢 问题解决了

**Reply by MaxMKMax** [reply]
@by2308333 您好, 不客气呢. 能帮到您我很开心! 我们才真的非常感谢您支持蓝灯. 有你这位用户是我们的荣幸. 谢谢!
